frontend/packages/console-shared/src/components/dashboard/dashboard-card/DashboardCard.tsx

```diff
@@ -15,7 +17,8 @@ const DashboardCard: React.FC<DashboardCardProps> = React.memo(
     >
       {children}
     </Card>
-  ),
+  )),
+  ErrorBoundaryFallback,
```

Comment:
I'm debating whether we should leave this off and just hide the card. That might be a better fallback in cases like this where the user simply doesn't have RBAC. We show the error in the console for development/debugging.

Reply:
I think that would be fine. I've removed the `ErrorBoundaryFallback` and also moved `withFallback` to `console-shared`